### ISSUE TITLE
Multiple bugs with Paksigue boss

### ISSUE THREAD

**Issue by swasun** [determinated/Aion-4.7.5-Rework-GiGatR00n-Source-Developing-Project#63]
- Geodata bug
- It reset if you attack it behing or in the corners of the citadel

![image](https://user-images.githubusercontent.com/25248913/65832142-8e451100-e2c0-11e9-9018-36adc49b861e.png)

**Comment by determinated** [comment]
I've tested here and are working normally. Maybe is how your geodata is set.

**Comment by swasun** [comment]
It's all set to true excepting the doors

**Comment by determinated** [comment]
To me is working normally
Maybe is something occasional

**Comment by swasun** [comment]
Yes it's not happening all the time, but it happened several times
